ci/tools/lookup-run-id

```diff
@@ -0,0 +1,99 @@
+#!/usr/bin/env bash
+
+# SPDX-FileCopyrightText: Copyright (c) 2025 NVIDIA CORPORATION & AFFILIATES. All rights reserved.
+#
+# SPDX-License-Identifier: Apache-2.0
```

Comment:
There is [this](https://github.com/nv-legate/cupynumeric.internal/blob/main/continuous_integration/scripts/tools/legate-gh-run-id) I wrote, based on [this](https://github.com/rapidsai/gha-tools/blob/main/tools/rapids-github-run-id) in rapids just to offer how other repos implemented.